ethereum/contracts/interfaces/relayer/IWormholeRelayer.sol

```diff
@@ -2,375 +2,580 @@
-        address relayProvider
-    ) external pure returns (uint256 receiverValue);
-
-    /**
-     * @notice Helper function that converts an EVM address to wormhole format
-     * @param addr (EVM 20-byte address)
-     * @return whFormat (32-byte address in Wormhole format)
-     */
-    function toWormholeFormat(address addr) external pure returns (bytes32 whFormat);
-
-    /**
-     * @notice Helper function that converts an Wormhole format (32-byte) address to the EVM 'address' 20-byte format
-     * @param whFormatAddress (32-byte address in Wormhole format)
-     * @return addr (EVM 20-byte address)
-     */
-    function fromWormholeFormat(bytes32 whFormatAddress) external pure returns (address addr);
-
-    /**
-     * @notice Returns the address of the current default relay provider
-     * @return relayProvider The address of (the default relay provider)'s contract on this source chain. This must be a contract that implements IRelayProvider.
-     */
-    function getDefaultRelayProvider() external view returns (address relayProvider);
-
-    /**
-     * @notice Returns default relay parameters
-     * @return relayParams default relay parameters
-     */
-    function getDefaultRelayParams() external pure returns (bytes memory relayParams);
-
-    /**
-     * @notice returns the address of the contract which delivers messages on this chain.
-     * I.E this is the address which will call receiveWormholeMessages.
-     */
-    function getDeliveryAddress() external view returns (address deliveryAddress);
-
-    error MsgValueMoreThanMaxAllowed(); // (maxTransactionFee, converted to target chain currency) + (receiverValue, converted to target chain currency) is greater than what your chosen relay provider allows
-    error MsgValueTooLow(); // msg.value is lower than the amount you specified (i.e., msg.value is less than (wormhole message fee) + (maxTransactionFee) + (receiverValue))
-    error MsgValueTooHigh(); // msg.value is higher than the amount you specified (i.e., msg.value is greater than (wormhole message fee) + (maxTransactionFee) + (receiverValue))
-    // Specifically, (msg.value) + (any leftover funds if this is a forward) is less than (maxTransactionFee + receiverValue), summed over all of your requests if this is a multichainSend/multichainForward
-    error MaxTransactionFeeNotEnough(); // maxTransactionFee is less than the minimum needed by your chosen relay provider
-    error NoDeliveryInProgress(); // Forwards can only be requested within execution of 'receiveWormholeMessages', or when a delivery is in progress
-    error ForwardRequestFromWrongAddress(); // A forward was requested from an address that is not the 'targetAddress' of the original delivery
-    error RelayProviderDoesNotSupportTargetChain(); // Your relay provider does not support the target chain you specified
+//TODO AMO: Decide on use regarding '' vs `` quotation of variables/function names in comments
+//            and then make it uniform.
```

Comment:
Comment on 'TODO AMO': Yes this would be nice. Probably not very high priority though